upgrade/upgrade-scripts/verify-jar

```diff
@@ -26,12 +26,22 @@ function usage() {
 	exit 2
 }
 
-function run_support_info() {
+function verify_jar_verify_cleanup() {
 	local rc="$?"
 
-	/opt/delphix/server/bin/support_info \
-		-o "${LOG_DIRECTORY}/${CONTAINER}" \
-		-t "log dropbox" || die "Failed to collect verify logs."
+	#
+	# This name is used by the "upgrade-verify.jar" just executed, so we
+	# cannot change this value without also modifying that JAR.
+	#
+	MDS_SNAPNAME="MDS-CLONE-upgradeverify"
+
+	[[ "$(/opt/delphix/server/bin/dx_manage_pg isrunning -s ${MDS_SNAPNAME})" -eq 0 ]] &&
+		(/opt/delphix/server/bin/dx_manage_pg stop -s "${MDS_SNAPNAME}" ||
+			die "failed to stop postgres for snapshot '${MDS_SNAPNAME}'.")
```

Comment:
This is spawning a subshell, right? I don't think `die` will cause the calling script to abort when it's used in a subshell. IIRC, it'll abort the subshell, but the calling process will continue.